results/15_RandomForest_LatentVar_TumorType.Rmd

```diff
@@ -0,0 +1,280 @@
+---
+title: "RF_latent_variable"
+author: "Jineta Banerjee"
+date: '`r format(Sys.time(), "%d %B, %Y")`'
+output: 
+  html_document:
+    toc: true
+    number_sections: true
+    toc_float:
+      collapsed: false
+      smooth_scroll: false
+    fig_width: 7
+    fig_height: 6
+    fig_caption: true
+    df_print: paged
+    code_folding: hide
+    
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+```{r libraries, echo=FALSE, eval=TRUE, results='hide', message=FALSE, warning=FALSE, include=FALSE}
+
+library(synapser)
+library(synapserutils)
+library(BiocManager)
+library(gProfileR)
+library(GOsummaries)
+library(tidyverse)
+library(DT)
+library(magrittr)
+library(colorspace)
+library(RColorBrewer)
+
+#Random Forest
+library(randomForest)
+library(e1071)
+library(caret)
+
+#plotting
+library(AppliedPredictiveModeling)
+transparentTheme(trans = .4)
+library(pheatmap)
+
+library(AnnotationDbi)
+#library(hgu95av2.db)
+#library(STRINGdb)
+library(gridExtra)
+
+#Synapser
+synLogin()
+```
+
+## Introduction
+
+This document describes training a random forest model using laten variables generated by transfer learning approaches. We are using the latent variables found in pNF and MPNST data to train the forest to find the most important classifiers for these two tumor types. We chose to train a random forest classifier to identify NF tumortypes for the following features of the model :
+* robustness to high dimensinality data
```

Comment:
these bullets don't render correctly - I think it might need an extra line before the first *

Reply:
fixed